catalystwan/models/configuration/feature_profile/sdwan/policy_object/color_list.py

```diff
@@ -0,0 +1,49 @@
+from enum import Enum
+from typing import List
+
+from pydantic import BaseModel, PrivateAttr
+
+from catalystwan.api.configuration_groups.parcel import Global, _ParcelBase
+from catalystwan.models.configuration.feature_profile.sdwan.policy_object.object_list_type import PolicyObjectListType
+
+
+class ColorType(str, Enum):
+    DEFAULT = "default"
+    MPLS = "mpls"
+    METRO_ETHERNET = "metro-ethernet"
+    BIZ_INTERNET = "biz-internet"
+    PUBLIC_INTERNET = "public-internet"
+    LTE = "lte"
+    THREEG = "3g"
+    RED = "red"
+    GREEN = "green"
+    BLUE = "blue"
+    GOLD = "gold"
+    SILVER = "silver"
+    BRONZE = "bronze"
+    CUSTOM1 = "custom1"
+    CUSTOM2 = "custom2"
+    CUSTOM3 = "custom3"
+    PRIVATE1 = "private1"
+    PRIVATE2 = "private2"
+    PRIVATE3 = "private3"
+    PRIVATE4 = "private4"
+    PRIVATE5 = "private5"
+    PRIVATE6 = "private6"
+
+
+class Color(Global):
+    value: ColorType
+
+
+class ColorEntry(BaseModel):
+    color: Color
+
+
+class ColorData(BaseModel):
+    entries: List[ColorEntry]
+
+
+class ColorPayload(_ParcelBase):
+    _payload_endpoint: PolicyObjectListType = PrivateAttr(default=PolicyObjectListType.COLOR)
```

Comment:
Coupling a model to endpoint as an attribute of model does not fit current approach.
This coupling should be done in `endpoint` definition.

Reply:
will create mapping